keith-vscode/src/kico/compilation-data-provider.ts

```diff
@@ -631,7 +631,7 @@ export class CompilationDataProvider implements vscode.TreeDataProvider<Compilat
      * Notifies the LS to cancel the compilation.
      */
     public async requestCancelCompilation(): Promise<void> {
-        this.lsClient.onReady().then(() => {
+        this.lsClient.start().then(() => {
```

Comment:
With what vscode version did this APi change or is onReady just wrong?

Reply:
The `vscode-langaugeclient` dependency update from version 5 to 9 included that change, which is documented that `onReady` should be replaced by `start` [here](https://github.com/microsoft/vscode-languageserver-node/blob/main/README.md#3170-protocol-800-json-rpc-800-client-and-800-server).